tests/Unit/ProtoTests/CustomLro/out/src/Gapic/CustomOperationResponse/Status.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * Generated by gapic-generator-php from the file
+ * https://github.com/googleapis/googleapis/blob/master/tests/Unit/ProtoTests/CustomLro/custom_lro.proto
+ * Updates to the above are reflected here through a refresh process.
+ */
+
+namespace Testing\CustomLro\Gapic\CustomOperationResponse;
```

Comment:
This looks good! my one suggestion: change the namespace to be `Enums` instead of `Gapic`

Reply:
Great idea. Done!